Client/Application/AppDelegate.swift

```diff
@@ -127,11 +128,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         navigationController.delegate = self
         navigationController.isNavigationBarHidden = true
 
-        if AppConstants.MOZ_STATUS_BAR_NOTIFICATION {
-            rootViewController = NotificationRootViewController(rootViewController: navigationController)
-        } else {
+//        if AppConstants.MOZ_STATUS_BAR_NOTIFICATION {
+//            rootViewController = NotificationRootViewController(rootViewController: navigationController)
+//        } else {
             rootViewController = navigationController
-        }
+//        }
```

Comment:
I prefer removing the comments and adding a comment with reference to a bugzilla, but at least add a comment as @sleroux suggests.